server/oneoff/add_buffs.py

```diff
@@ -0,0 +1,61 @@
+from app import session_scope
```

Comment:
Do you think you could change this to be `sync_buffs` instead of `add_buffs` so we can use it whenever buffs change?